AjobK/Athena
# 389 post timeout

closes #389 

<br />

 **What is it supposed to do**  
Adds a timeout for users trying to spam posts.

Used the branch from issue #386 to get a toast when someone tries to spam posts, duration of the timeout can be updated in the .env file
![image](https://user-images.githubusercontent.com/45096509/163716693-f4db300a-4fe7-491d-93cb-304c027ba072.png)

<br />

 **How can we test your branch**  
Create a post and try to create another post directly after. This should display an error and not create the post.

<br />

 **Additional notes**  
UPDATE THE ENV FILE

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
